tests/validation/tests/v3_api/resource/terraform/aks/aks.tf

```diff
@@ -11,7 +11,7 @@ resource "random_string" "random" {
 }
 
 resource "azurerm_resource_group" "default" {
-  name     = "testautoaks${random_string.random.result}"
+  name     = "testautoaks"
```

Comment:
Would there be a problem with multiple clusters in the same resource group?

Reply:
I'll double check this and get back to you @bmdepesa

Reply:
You're correct i reverted the resource name with a random suffix for now. Will figure out something for the delete ha for these two AKS/GKE cluster types later.